stytra/core.py

```diff
@@ -88,7 +87,7 @@ class Stytra:
                 for closed-loop experiments: either "vigor" for embedded experiments
                     or "position" for freely-swimming ones. A custom estimator can be supplied.
 
-        recording : bool (False) or dict
+        recording : dict
```

Comment:
So now recording always has to be specified?

Reply:
Yes, I judged it better to be explicit here. It saves debugging and should be clearer to the user (at least I have found myself in the situation where I tried both using a bool and a dict when things didn't work before). However, using a boolean should not be a problem if that is the preferred way to go.

Reply:
Annotate with Optional[Dict[str, Any]] as below please